_docs/overview/key-concepts.md

```diff
@@ -21,31 +21,24 @@ there is no legal recourse in the event of the key falling into the
 hands of someone else. Bitcoin is decentralized, meaning there is no
 company, person or entity that is liable.
 
-## Offline key storage ("cold storage")
+## Offline key storage
 
-You don't want to store your
-private key on any computer that's connected to the Internet ("hot
-storage"), because that exposes it to more hacking attempts. There are
-viruses out there that search computers for private keys and steal them
-(thereby stealing your money).
+Private keys should only ever be offline, or "in cold storage". They should
+never be stored on any internet-connected computer, or "in hot storage",
```

Comment:
I don't think the word "in" belongs inside the quotes. Cold storage and hot storage are the commonly used terms, often without including the word in.

Reply:
Corrected in a new commit.